convert2rhel/utils/files.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright(C) 2024 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+
+import errno
+import os
+
+
+def mkdir_p(path):
+    """Create all missing directories for the path and raise no exception
+    if the path exists.
+    """
+    try:
+        os.makedirs(path)
+    except OSError as err:
+        if err.errno == errno.EEXIST and os.path.isdir(path):
+            pass
+        else:
+            raise
```

Comment:
Maybe we could rewrite the function to remove the else part like this? 

```suggestion
        if err.errno == errno.EEXIST and os.path.isdir(path):
            return
         
         raise
```

Reply:
Sure LGTM. Didn't change any code with this PR, just moved things around